spanner/api/Spanner.Samples.Tests/InsertUsingDmlReturningAsyncPostgreTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class InsertUsingDmlReturningAsyncPostgreTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public InsertUsingDmlReturningAsyncPostgreTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestInsertUsingDmlReturningAsyncPostgre()
```

Comment:
Can we rename everything that currently ends with "Postgre" to "Postgres"? While it's unfortunate that "PostgreSQL" has a compound name, when it's referred to without the "SQL" part, [it's as "Postgres" rather than "Postgre"](https://en.wikipedia.org/wiki/PostgreSQL).

Reply:
Hi Jon, It's consistent with the existing naming convention. I'll raise another PR after this to update all `Postgre` to `Postgres`, if you agree ?

Reply:
Yes, I guess. It's a pity that it got this far :(